### DISCUSSION TITLE
Eaglercraft kirby

### DISCUSSION THREAD

**Discussion by lil1Toni93** [3kh0/ext-remover#518] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Resent crashed! Please dm hooman#1196 about this crash. If you're launching for the first time, don't worry! Just refresh and you'll be fine! Github: https://github.com/ThisIsALegitUsername/Resent-web

Native Browser Exception
----------------------------------
  Line: https://3kh0.github.io/files/resent-client/classes.js:32:97
  Type: TypeError
  Desc: Cannot read properties of undefined (reading 'length')
----------------------------------

TypeError: Cannot read properties of undefined (reading 'length')
    at $rt_str (https://3kh0.github.io/files/resent-client/classes.js:32:97)
    at Bp9 (https://3kh0.github.io/files/resent-client/classes.js:17546:64)
    at BbD (https://3kh0.github.io/files/resent-client/classes.js:17547:19)
    at A.AGP (https://3kh0.github.io/files/resent-client/classes.js:17548:21)
    at $rt_array.toString (https://3kh0.github.io/files/resent-client/classes.js:19597:352)
    at Worker.fn (https://3kh0.github.io/files/resent-client/classes.js:2541:87)

eaglercraft.version = "22w43a"
eaglercraft.username = "Creeper"

webgl.version = WebGL 2.0 (OpenGL ES 3.0 Chromium)
webgl.renderer = Mesa DRI Intel(R) UHD Graphics 600 (GLK 2)
webgl.vendor = Intel Open Source Technology Center

webgl.anisotropicGlitch = false

window.eaglercraftOpts = {"container":"game_frame","assetsURI":"packs/kirby.epk","serverWorkerURI":"worker_bootstrap.js","worldsFolder":"MAIN","servers":[{"serverName":"Blobcraft","serverAddress":"wss://blobcraft.minecraft.pe","hideAddress":true},{"serverName":"Asspixel","serverAddress":"wss://web.asspixel.net/CAP/","hideAddress":true},{"serverName":"VanillaCraft","serverAddress":"wss://play.vanillacraftsmp.org","hideAddress":true},{"serverName":"CraftClue","serverAddress":"wss://play.craftclue.rocks","hideAddress":true},{"serverName":"Ayunboom","serverAddress":"wss://sus.shhnowisnottheti.me","hideAddress":true},{"serverName":"Sealcraft","serverAddress":"wss://sealcraft.ddns.net:442/servers","hideAddress":true},{"serverName":"MessCraft","serverAddress":"wss://mess.eu.org/s1/","hideAddress":true},{"serverName":"IMC.RE","serverAddress":"wss://play.imc.re:441/server","hideAddress":true},{"serverName":"HardCoreSMP","serverAddress":"wss://hardcoresmp.mrnewt.repl.co/serve","hideAddress":true}],"relays":[{"addr":"wss://relay.deev.is/","name":"lax1dude relay #1","primary":false},{"addr":"wss://relay.lax1dude.net/","name":"lax1dude relay #2","primary":false},{"addr":"wss://relay.shhnowisnottheti.me/","name":"ayunami relay #1","primary":true}],"mainMenu":{"splashes":["Press Y to open GUI!"],"eaglerLogo":false}}

window.navigator.userAgent = Mozilla/5.0 (X11; CrOS x86_64 14541.0.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36
window.navigator.vendor = Google Inc.
window.navigator.language = en
window.navigator.hardwareConcurrency = 2
window.navigator.deviceMemory = 4
window.navigator.platform = Linux x86_64
window.navigator.product = Gecko

rootElement.clientWidth = 1470
rootElement.clientHeight = 750
window.innerWidth = 1470
window.innerHeight = 750
window.outerWidth = 1469
window.outerHeight = 853
window.devicePixelRatio = 0.8999999761581421
window.screen.availWidth = 1469
window.screen.availHeight = 853
window.screen.colorDepth = 24
window.screen.pixelDepth = 24

window.location.href = https://3kh0.github.io/files/resent-client/index.html#

----- Begin Minecraft Config -----
CgAAAwAFQ1BTX3kAAAAEAwAUSGVhbHRoIERpc3BsYXlfbGFzdFgAAAAACQAGcmVsYXlzCgAAAAYIAARhZGRyAAxAZWFnbGVyY3JhZnQBAAdwcmltYXJ5AQgAB2NvbW1lbnQAC0VhZ2xlcmNyYWZ0AAgABGFkZHIACUJsb3BDcmFmdAEAB3ByaW1hcnkACAAHY29tbWVudAAPUmVsYXkgU2VydmVyICM2AAgABGFkZHIAE3dzczpjcmVlcGVyX2FzYWVsLC8BAAdwcmltYXJ5AAgAB2NvbW1lbnQACmFzYWVsIGxhbmQACAAEYWRkcgAZd3NzOi8vcmVsYXkubGF4MWR1ZGUubmV0LwEAB3ByaW1hcnkACAAHY29tbWVudAARbGF4MWR1ZGUgcmVsYXkgIzIACAAEYWRkcgAUd3NzOi8vcmVsYXkuZGVldi5pcy8BAAdwcmltYXJ5AAgAB2NvbW1lbnQAEWxheDFkdWRlIHJlbGF5ICMxAAgABGFkZHIAIHdzczovL3JlbGF5LnNoaG5vd2lzbm90dGhldGkubWUvAQAHcHJpbWFyeQAIAAdjb21tZW50ABBheXVuYW1pIHJlbGF5ICMxAAMACGtleS56b29tAAAALgMAFEhlYWx0aCBEaXNwbGF5X2xhc3RZAAAAAAEAFU5vUGFydGljbGVzX2lzRW5hYmxlZAABAA9jaGF0TGlua3NQcm9tcHQBAQAUQW5pbWF0aW9uc19pc0VuYWJsZWQACAArS2V5c3Ryb2tlc19Nb2RlU2V0dGluZ19QcmVzc2VkIGJ1dHRvbiBjb2xvcgAFV2hpdGUBABBwYXRjaEFuaXNvdHJvcGljAAEAJENvbWJvQ291bnRlcl9Cb29sU2V0dGluZ19UZXh0IFNoYWRvdwEBABZTbW9vdGhDYW1lcmFfaXNFbmFibGVkAAEAGkF1dG9HR19Cb29sU2V0dGluZ19PbiBMb3NlAQEAEE5vUmFpbl9pc0VuYWJsZWQABQARdm9pY2VMaXN0ZW5Wb2x1bWU/AAAAAQALc2hvd1NraW5IYXQAAwAOUmVhY2hEaXNwbGF5X3gAAAAEAQASVG9vbHRpcHNfaXNFbmFibGVkAAMADlJlYWNoRGlzcGxheV95AAAAIgEACmR5bmFtaWNmb3YBAQAIc2hvd1JhaW4BAwASUmVhY2hEaXNwbGF5X2xhc3RYAAAAAAMAElJlYWNoRGlzcGxheV9sYXN0WQAAAAADAAtrZXkuZm9yd2FyZAAAABEDAAhrZXkubGVmdAAAAB4DAAxQb3RDb3VudGVyX3gAAAAEAwAMUG90Q291bnRlcl95AAAAQAMAEFBvdENvdW50ZXJfbGFzdFgAAAAAAwAQUG90Q291bnRlcl9sYXN0WQAAAAABABZQaW5nIERpc3BsYXlfaXNFbmFibGVkAAEAJktleXN0cm9rZXNfQm9vbFNldHRpbmdfUk1CIGNwcyBjb3VudGVyAQEAFUF1dG9SZXNwYXduX2lzRW5hYmxlZAADAA5saW1pdEZyYW1lcmF0ZQAAAAABABRzZWVuQWNrbm93bGVkZ2VtZW50cwEBABRLZXlzdHJva2VzX2lzRW5hYmxlZAADAAhrZXkuanVtcAAAADkBACJQb3RDb3VudGVyX0Jvb2xTZXR0aW5nX1RleHQgU2hhZG93AQEAFkNvbWJvQ291bnRlcl9pc0VuYWJsZWQAAQAWVG9nZ2xlU3ByaW50X2lzRW5hYmxlZAABABZSZWFjaERpc3BsYXlfaXNFbmFibGVkAAEACmZ1bGxCcmlnaHQBAQALdG91Y2hzY3JlZW4AAQAHaGlkZVRNSQEBAAtpbnZlcnRNb3VzZQABAA5zaG93U2NvcmVib2FyZAEBAAxzbW9vdGhDYW1lcmEBAQANZmFuY3lHcmFwaGljcwABABNXYXRlcm1hcmtfaXNFbmFibGVkAAEAE1BvdGlvbkhVRF9pc0VuYWJsZWQAAQAUUG90Q291bnRlcl9pc0VuYWJsZWQAAQAURnVsbEJyaWdodF9pc0VuYWJsZWQAAwAFRlBTX3gAAAAEAwAFRlBTX3kAAAAYCAApS2V5c3Ryb2tlc19Nb2RlU2V0dGluZ19QcmVzc2VkIHRleHQgY29sb3IABUJsYWNrAQAOSW5mb19pc0VuYWJsZWQAAwAMa2V5LmZ1bmN0aW9uAAAAIwMACUNQU19sYXN0WAAAAAADAAprZXkuc3ByaW50AAAAEwMACUNQU19sYXN0WQAAAAADAA5jaGF0VmlzaWJpbGl0eQAAAAABABBzaG93U2tpblJpZ2h0TGVnAQMADWtleS5pbnZlbnRvcnkAAAASAQASQXJtb3JIdWRfaXNFbmFibGVkAAEAGFNib2FyZCBudW1iZXJzX2lzRW5hYmxlZAABABNOb0h1cnRDYW1faXNFbmFibGVkAAMAElBpbmcgRGlzcGxheV9sYXN0WAAAAAADABJQaW5nIERpc3BsYXlfbGFzdFkAAAAAAQAVc2hvd1Njb3JlYm9hcmROdW1iZXJzAAMAEEtleXN0cm9rZXNfbGFzdFgAAAAABQALc2Vuc2l0aXZpdHk++MnqAwAQS2V5c3Ryb2tlc19sYXN0WQAAAAABAAdzaG93RlBTAQMACXBhcnRpY2xlcwAAAAADAAZTbG90IDEAAAACAwAGU2xvdCAyAAAAAwMABlNsb3QgMwAAAAQDAAZTbG90IDQAAAAFAwAIQ2xpY2tHdWkAAAAnAwAGU2xvdCA1AAAABgMABlNsb3QgNgAAAAcDAAZTbG90IDcAAAAIAQAYSGVhbHRoIERpc3BsYXlfaXNFbmFibGVkAAMABlNsb3QgOAAAAAkDAAprZXkuYXR0YWNr////nAMABlNsb3QgOQAAAAoBAA9zaG93Q29vcmRpbmF0ZXMBAQADZm9nAAEADUZQU19pc0VuYWJsZWQAAQAbQ1BTX0Jvb2xTZXR0aW5nX1RleHQgc2hhZG93AQEAEkZyZWVMb29rX2lzRW5hYmxlZAADAAlrZXkuc25lYWsAAAAqAwAPUG90aW9uSFVEX2xhc3RYAAAAAAEAJktleXN0cm9rZXNfQm9vbFNldHRpbmdfTE1CIGNwcyBjb3VudGVyAQMAD1dhdGVybWFya19sYXN0WAAAAAADAA9Qb3Rpb25IVURfbGFzdFkAAAAAAwAPV2F0ZXJtYXJrX2xhc3RZAAAAAAEAG0tleXN0cm9rZXNfQm9vbFNldHRpbmdfSnVtcAEFAANmb3Y/A5sLAwAIZ3VpU2NhbGUAAAACBQAJY2hhdFNjYWxlP4AAAAEAEGhlbGRJdGVtVG9vbHRpcHMBAQAQc2hvd1NraW5SaWdodEFybQEDAA5Bcm1vckh1ZF9sYXN0WAAAAAADAA5Bcm1vckh1ZF9sYXN0WQAAAAABAAZjbG91ZHMABQATY2hhdEhlaWdodFVuZm9jdXNlZD7jJ6kIAC1LZXlzdHJva2VzX01vZGVTZXR0aW5nX1VucHJlc3NlZCBidXR0b24gY29sb3IABUJsYWNrAQAaQXJtb3JIdWRfQm9vbFNldHRpbmdfQm9vdHMBAQAFZ2xpbnQBAQAOcmVuZGVyVmlnbmV0dGUBAwANYW50aWFsaWFzTW9kZQAAAAEFABB2b2ljZVNwZWFrVm9sdW1lPwAAAAEAFENsZWFyIENoYXRfaXNFbmFibGVkAAUAC3NvdW5kVm9sdW1lP4AAAAMADlBpbmcgRGlzcGxheV94AAAABAEAD3Nob3dTa2luTGVmdExlZwEDAA5QaW5nIERpc3BsYXlfeQAAAEoBABFoaWRlU2VydmVyQWRkcmVzcwABAA5hbGxvd0ZOQVdTa2lucwEBAAt2aWV3Qm9iYmluZwABABBhZHZhbmNlZFRvb2x0aXBzAAMADEtleXN0cm9rZXNfeAAAABkDAAhrZXkuYmFjawAAAB8DAAxLZXlzdHJva2VzX3kAAAAEAwAIa2V5LmRyb3AAAAAQAQAQcGF1c2VPbkxvc3RGb2N1cwEDAAxGYWtlIEhhY2tzX3gAAAMgAwALdm9pY2VQVFRLZXkAAAAvAwAMRmFrZSBIYWNrc195AAAABAMADkNvbWJvQ291bnRlcl94AAAABAMADlRvZ2dsZVNwcmludF94AAAABAMADkNvbWJvQ291bnRlcl95AAAABAMADlRvZ2dsZVNwcmludF95AAAACgMAEEZha2UgSGFja3NfbGFzdFgAAAAAAwAIRnJlZWxvb2sAAAAmAwAQRmFrZSBIYWNrc19sYXN0WQAAAAABAApjbGVhckdsYXNzAAEAIktleXN0cm9rZXNfQm9vbFNldHRpbmdfVHJhbnNwYXJlbnQAAwALV2F0ZXJtYXJrX3gAAAEsAwALUG90aW9uSFVEX3gAAAAEAwALV2F0ZXJtYXJrX3kAAAAEAQANQ1BTX2lzRW5hYmxlZAADAAtQb3Rpb25IVURfeQAAAV4BAAljaGF0TGlua3MBAQAPc21hcnREaXNjb25uZWN0AQMABkluZm9feAAAAAQBABZOb0R5bmFtaWNGT1ZfaXNFbmFibGVkAAMABkluZm9feQAAAA4BAB1Bcm1vckh1ZF9Cb29sU2V0dGluZ19MZWdnaW5ncwEBABtBcm1vckh1ZF9Cb29sU2V0dGluZ19IZWxtZXQBBQAJY2hhdFdpZHRoP4AAAAMACkFybW9ySHVkX3gAAAAyAwAKQXJtb3JIdWRfeQAAAAQBACJLZXlzdHJva2VzX0Jvb2xTZXR0aW5nX1RleHQgc2hhZG93AAgACGxhbmd1YWdlAAVlbl9VUwEAD3Nob3dTa2luTGVmdEFybQEBABRGYWtlIEhhY2tzX2lzRW5hYmxlZAABABBBdXRvR0dfaXNFbmFibGVkAAEAGU1pbmltYWwgQm9iYmluZ19pc0VuYWJsZWQAAQAkUGluZyBEaXNwbGF5X0Jvb2xTZXR0aW5nX1RleHQgU2hhZG93AQEAEkF1dG9XYWxrX2lzRW5hYmxlZAABAAhzaG93Q2FwZQEBABZDaHVua0JvcmRlcnNfaXNFbmFibGVkAAMAEEhlYWx0aCBEaXNwbGF5X3gAAAAEAwAHa2V5LnVzZQAAACEDABBIZWFsdGggRGlzcGxheV95AAAAIggACmxhc3RTZXJ2ZXIACUJsb2JjcmFmdAMADGtleS5waWNrSXRlbQAAABUDAAhrZXkuY2hhdAAAABQBAA5zaG93U2tpbkphY2tldAEDABF2b2ljZUxpc3RlblJhZGl1cwAAABADAAlGUFNfbGFzdFgAAAAAAwAJRlBTX2xhc3RZAAAAAAEADHJlbGF5VGltZW91dA8JAAdzZXJ2ZXJzCgAAAAkBAAtoaWRlQWRkcmVzcwAIAAJpcAACaGkIAARuYW1lAAcgU2VydmVyAAEAC2hpZGVBZGRyZXNzAQgAAmlwAARIb2xhCAAEbmFtZQADSHViAAEAC2hpZGVBZGRyZXNzAAgAAmlwAAVIZWxsbwgABG5hbWUADEFzYWVsIFNlcnZlcgABAAtoaWRlQWRkcmVzcwAIAAJpcAAOM2toby5naXRodWIuaW8IAARuYW1lABBNaW5lY3JhZnQgU2VydmVyAAEAC2hpZGVBZGRyZXNzAAgAAmlwABV3c3M6Ly9tZXNzLmV1Lm9yZy9zMS8IAARuYW1lAAltZXNzY3JhZnQAAQALaGlkZUFkZHJlc3MACAACaXAAEkdPQUwgSVMgTk9UIFRPIERJRQgABG5hbWUABk15IFNNUAABAAtoaWRlQWRkcmVzcwAIAAJpcAAKZiBuamRyZm5mbggABG5hbWUAEE1pbmVjcmFmdCBTZXJ2ZXIAAQALaGlkZUFkZHJlc3MACAACaXAAC0BlYWRlcmNyYWZ0CAAEbmFtZQALRWFnbGVyY3JhZnQAAQALaGlkZUFkZHJlc3MACAACaXAAATgIAARuYW1lABBNaW5lY3JhZnQgU2VydmVyAAUAC2NoYXRPcGFjaXR5P4AAAAEAEkF1dG9KdW1wX2lzRW5hYmxlZAABAB9Bcm1vckh1ZF9Cb29sU2V0dGluZ19DaGVzdHBsYXRlAQEAGkluZm9fQm9vbFNldHRpbmdfRGlyZWN0aW9uAQEADGhpZGVKb2luQ29kZQABAB5LZXlzdHJva2VzX0Jvb2xTZXR0aW5nX1JhaW5ib3cAAwAKZm9yY2VXaWR0aAAAAAAIACtLZXlzdHJva2VzX01vZGVTZXR0aW5nX1VucHJlc3NlZCB0ZXh0IGNvbG9yAAVXaGl0ZQMADGNodW5rVXBkYXRlcwAAAAQBABZOb1N3aW5nRGVsYXlfaXNFbmFibGVkAAUAC211c2ljVm9sdW1lAAAAAAEAI1RvZ2dsZVNwcmludF9Cb29sU2V0dGluZ19UZXh0IERyYXduAQEAE0ZwcyBCb29zdF9pc0VuYWJsZWQABQAFZ2FtbWE/gAAAAQAOc2hvd090aGVyQ2FwZXMAAQALY2hhdENvbG91cnMBAwASQ29tYm9Db3VudGVyX2xhc3RYAAAAAAMAElRvZ2dsZVNwcmludF9sYXN0WAAAAAADABJUb2dnbGVTcHJpbnRfbGFzdFkAAAAAAwASQ29tYm9Db3VudGVyX2xhc3RZAAAAAAgAGkhpdGJveGVzX01vZGVTZXR0aW5nX0NvbG9yAAVXaGl0ZQMADHZpZXdEaXN0YW5jZQAAAAMBAApkaWZmaWN1bHR5AQMACWtleS5yaWdodAAAACADAApJbmZvX2xhc3RYAAAAAAMACkluZm9fbGFzdFkAAAAAAQAIYWRkZXJhbGwAAQAWU2VsZiBOYW1ldGFnX2lzRW5hYmxlZAADABBhbWJpZW50T2NjbHVzaW9uAAAAAggAG0tleXN0cm9rZXNfTW9kZVNldHRpbmdfU2l6ZQAFU21hbGwFABFjaGF0SGVpZ2h0Rm9jdXNlZD+AAAABAAhhbmFnbHlwaAABAApmdWxsU2NyZWVuAAEAEkhpdGJveGVzX2lzRW5hYmxlZAABABlBdXRvR0dfQm9vbFNldHRpbmdfT24gV2luAQEAFFNjb3JlYm9hcmRfaXNFbmFibGVkAAEAJEF1dG9HR19Cb29sU2V0dGluZ19SZXBldGl0aW9uIGJ5cGFzcwEDAA5rZXkucGxheWVybGlzdAAAABkDAAtmb3JjZUhlaWdodAAAAAABABlBcm1vckh1ZF9Cb29sU2V0dGluZ19JdGVtAQEAHEtleXN0cm9rZXNfQm9vbFNldHRpbmdfU25lYWsAAwAFQ1BTX3gAAAAyAA==
----- End Minecraft Config -----

window.minecraftServer = undefined

**Comment by lil1Toni93** [comment]
can yall help me

**Reply by rwg100** [reply]
no

**Comment by rwg100** [comment]
penis balls i bounced on jumping joe 34 times in ur moms house boing boing mcboing loing loing i ate a burger king whopper whopper whopper whopper junior double triple whopper flame grilled something i dont know the rest penis mcfuckfaceinukednorthkorabyeeee

**Comment by obkduishinijf** [comment]
@lil1Toni93 
1. you ain't gotta post the whole error
2. try again, if it is your first time, then refresh
